src/writer/creator.cpp

```diff
@@ -381,8 +381,7 @@ namespace zim
       while (true) {
          auto r = read(out_fd, batch_read, 1024);
          if (r == -1) {
-           perror("Cannot read");
-           throw std::runtime_error("oups");
+           throw std::runtime_error(std::strerror(errno));
```

Comment:
Why don't you move this commit ("Make zim::writer::Creator thrown better exception.") earlier in history?

Reply:
Done